packages/venia-concept/src/reducers/app.js

```diff
@@ -7,7 +7,8 @@ export const name = 'app';
 const initialState = {
     drawer: null,
     overlay: false,
-    pending: {}
+    pending: {},
+    isOnline: navigator.onLine
```

Comment:
I saw earlier that the `App` component was keeping local state for whether it had ever been online. Why would we not put that here in redux state?

Reply:
Good point, I'll look into adding it.